rest_framework/relations.py

```diff
@@ -71,8 +72,8 @@ def __str__(self):
 
 class RelatedField(Field):
     queryset = None
-    html_cutoff = 1000
-    html_cutoff_text = _('More than {count} items...')
+    html_cutoff = int(api_settings.HTML_SELECT_CUTOFF)
+    html_cutoff_text = _(api_settings.HTML_SELECT_CUTOFF_TEXT)
```

Comment:
Setting these defaults in `__init__` would be better, as we're trying to avoid accessing settings until runtime.

Some subtleties around how best to do that, as we'd still want to preserve any value that *has* been set as a class attribute tho (eg user has a custom relational field, where they've overridden this value)